lib/pbkit/pbkit.c

```diff
@@ -3349,33 +3349,33 @@ int pb_init(void)
     *(p++)=12;
     pb_push(p++,NV20_TCL_PRIMITIVE_3D_ACTIVATE_COLORS,1);
```

Comment:
I noticed this on by accident - looks like it could be `pb_push1` (as part of commit `pbkit: Use new pb_push* in pbkit functions`)? The one before this, too.
However, also acceptable to do this in a full pbkit refactor (as it's easy to introduce errors again).

Reply:
Good catch. I believe I had touched these two in an earlier attempt. They're the only ones using the same command and having a "standard" number of parameters not using one of the helper functions. Ought to be fixed now. :+1: